src/components/axes/categoryAxis.ts

```diff
@@ -137,7 +137,7 @@ export module Axis {
      */
     private _measureTicks(axisWidth: number, axisHeight: number, scale: Scale.Ordinal, ticks: string[]) {
       var wrappingResults = ticks.map((s: string) => {
-        var bandWidth = scale.fullBandStartAndWidth(s)[1];
+        var bandWidth = scale.rangeBand();
```

Comment:
I think this change breaks something, becuase `fullBandAndStartWidth(s)[1]` is wider than `rangeBand()`